awesome_dashboard/static/src/dashboard/numbercard/numbercard.js

```diff
@@ -0,0 +1,9 @@
+import { Component } from "@odoo/owl";
+
+export class NumberCard extends Component {
+    static template = "awesome_dashboard.NumberCard";
+    static props = {
+        title: { type: String },
+        value: { type: Number },
```

Comment:
```suggestion
        title: String,
        value: Number,
```
If you're only passing type no need to make it an object